Formula/trurl.rb

```diff
@@ -0,0 +1,20 @@
+class Trurl < Formula
+  desc "Command-line tool for URL parsing and manipulation"
+  homepage "https://github.com/curl/trurl"
+  url "https://github.com/curl/urler/archive/refs/tags/trurl-0.2.tar.gz"
```

Comment:
Sorry, my bad!

```suggestion
  url "https://github.com/curl/trurl/archive/refs/tags/trurl-0.2.tar.gz"
```

Reply:
actually that link works as a redirect. I did test download and re-checked checksum before applying it. 

```sh
$ wget https://github.com/curl/urler/archive/refs/tags/trurl-0.2.tar.gz
--2023-04-03 14:19:06--  https://github.com/curl/urler/archive/refs/tags/trurl-0.2.tar.gz
Resolving github.com (github.com)... 140.82.113.3
Connecting to github.com (github.com)|140.82.113.3|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://github.com/curl/trurl/archive/refs/tags/trurl-0.2.tar.gz [following]
--2023-04-03 14:19:08--  https://github.com/curl/trurl/archive/refs/tags/trurl-0.2.tar.gz
Reusing existing connection to github.com:443.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/curl/trurl/tar.gz/refs/tags/trurl-0.2 [following]
--2023-04-03 14:19:08--  https://codeload.github.com/curl/trurl/tar.gz/refs/tags/trurl-0.2
```

Reply:
Yea, that should work, too. But let's use the new name here.